XbSymbolDatabase.h

```diff
@@ -551,6 +951,73 @@ void XbSymbolContinuousSigScan(bool enable);
 /// <param name="enable">Input boolean to use first symbol address only or not.</param>
 void XbSymbolFirstDetectAddressOnly(bool enable);
 
+// ******************************************************************
+// * API v3 (context) functions to use with other projects.
```

Comment:
Why even keep the old API? People who don't want to use the new API can just check out the old version (at loss of features), or make independent wrappers.
Porting apps is probably easy eitherway (and there aren't many users to begin with).

Reply:
Updated in the comment section.